db-tests/src/test/java/gov/nasa/jpl/aerie/database/PlanCollaborationTests.java

```diff
@@ -594,24 +595,26 @@ void snapshotInheritsAllLatestAsParents() throws SQLException{
 
       //take n snapshots, then insert them all into the latest table
       final int numberOfSnapshots = 4;
-      final int[] snapshotIds = new int[numberOfSnapshots];
-      for(int i = 0; i < numberOfSnapshots; ++i){
-        snapshotIds[i] = createSnapshot(planId);
+      final var snapshotIds = new ArrayList<Integer>(numberOfSnapshots);
+      for(int i = 0; i < numberOfSnapshots; i++){
+        snapshotIds.add(createSnapshot(planId));
```

Comment:
Why was `snapshotIds` changes from an int[] to an ArrayList?

Reply:
My intent here was to be able to directly compare the two lists for equality - in the event that the assertion fails, it provides both lists in the message, rather than just the element that was unequal.

https://github.com/NASA-AMMOS/aerie/blob/63bb374ca4346d9c5ed5390b3e8e8a8c30c5b2dc/db-tests/src/test/java/gov/nasa/jpl/aerie/database/PlanCollaborationTests.java#L652